### PR TITLE
Chart fixes and en wiki nvd3 chart export

### PR DESCRIPTION
**Pull request by Groovman**
Some fixes for charts translation and chart export for en wiki nvd3chart format export.

----
- Add support for export nvd3 chart export.
image

- Give 2nd skill name of Beast Weapons(四象武器) when naming.
e.g. Baihu Claw **Pontus**.

### CODE REVIEW

File: src/translate.js [hunk 1]
```diff
@@ -1754,7 +1759,7 @@ var multiLangData = {
         "zh": "加入图表",
     },
     "本": {
-        "en": "",
+        "en": "x",
```
Comment (kagekiyo7) [inline src/translate.js]: There are three places use key of "本". Is there no problem? (just confirm)

image
image
image

File: src/chart.js [hunk 2]
```diff
@@ -169,9 +235,15 @@ var HPChart = CreateClass({
                             {selector[locale].supported_chartsortkeys}
                         </FormControl>
                         <span>CSV</span>
-                        <CSVLink className={"btn btn-default"} data={csvData}><span
+                        <CSVLink className={"btn btn-default"} data={csvData} filename={"motocal_charts.csv"}><span
+                            className={"glyphicon glyphicon-download-alt"} aria-hidden={"true"}></span></CSVLink>
+                        <button className={"btn btn-default"}><CopyToClipboard text={csvClipBoardData}><span
+                            className={"glyphicon glyphicon-copy"} aria-hidden={"true"}></span></CopyToClipboard>
+                        </button>
+                        <span>WIKI</span>
```
Comment (kagekiyo7) [inline src/chart.js]: "WIKI" should change to "NVD3" to make it more specific. (japanese wiki can't use this)

File: src/translate.js (hunk 1 above)
Comment (Groovman) [reply src/translate.js]: Yeah set names in chart section are looking strange with just a number at the end and 1x would be one time in western same as  1本.

File: src/chart.js (hunk 2 above)
Comment (Groovman) [reply src/chart.js]: Done

**Comment by kagekiyo7** [comment]
I added a description without permission on top.

**Comment by Groovman** [comment]
Technically already got used https://gbf.wiki/User:Midokuni/Notepad/MotocalExport/Colo :)

**Pull request merged**